src/main/java/de/tarent/entities/Category.java

```diff
@@ -10,6 +11,7 @@
 @Entity
 @EqualsAndHashCode
 @ToString
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Why should we use this? This is bad form in general.

Reply:
Why shouldn't we use this, it makes the application more robust against client errors

Reply:
Silently ignoring errors is bad form.
If we want to do it though it should be configured globally (by changing the ObjectMapper defaults) instead of on each class.

Reply:
It's not about ignoring errors, the application simply does not care about additional properties the client sends.
Regarding the configuration I had the same idea yesterday and moved it to JacksonObjectMapperCustomizer